topics/xamarin/index.md

```diff
@@ -0,0 +1,20 @@
+---
+aliases: mono
```

Comment:
Maybe this should be a related topic, not an alias? (unless Mono and Xamarin are exactly the same)

Reply:
Moved it as related.